pom.xml

```diff
@@ -974,6 +974,7 @@
                             <exclude>**/*.md</exclude>
                             <exclude>.java-version</exclude>
                             <exclude>.python-version</exclude>
+                            <exclude>systemvm/.pythen-version</exclude>
```

Comment:
@DaanHoogland typo?

Reply:
yes,m definitely,  but also this file is not in use anymore.